README.md

```diff
@@ -0,0 +1,85 @@
+# Zerobase API Server
+
+This repository contains the back end for the Zerobase smart tracing platform. Refer to the [smart-tracing repo](https://github.com/zerobase-io/smart-tracing) for the front end. We are using Kotlin and Neo4j with cloud storage on Heroku. We are using Dropwizard for the REST framework. Kotlin requires Java and Maven.
+
+# TODO
+
+*
+
+## Set up for the project
+
+### Kotlin
+
+* The backend is written in Kotlin. Make sure you have a compatible IDE such as [IntelliJ](https://www.jetbrains.com/idea/download/index.html?_ga=2.137859766.761208892.1584829709-1795868819.1584829709#section=mac).
+
+
+### Java 11
+
+* This project requires Java version 11 which can be dowloaded [here](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+### Maven
```

Comment:
these are best installed with a package manager, so we probably want a comment about installing that, with `brew` for mac